CONTRIBUTING.md

```diff
@@ -24,6 +24,8 @@ We are currently managing payment via Upwork. If you'd like to be paid for your
 
 ## Submitting a Pull Request
 #### Proposing a Change
+1. *Before writing any code* write a proposal for the solution in the issue
+1. Wait for Expensify to review and provide feedback on the proposal within the GH issue, and assign the GH issue to you
```

Comment:
I'd also mention here `Please do not move forward with creating a Pull Request until your proposal is approved`